src/components/App/App.jsx

```diff
@@ -36,18 +35,13 @@ export default function App() {
     quantity: ""
   }
 
-  //TODO: replace these placeholders once we integrate with the recipe API
-  const sampleRecipeInstructions = constRecipeInstructions
-
-  const sampleRecipeInfo = constRecipeInfo
-
   const recipeApiUrl = "http://localhost:3001/apirecipes/"
```

Comment:
nit: maybe rename these to `listRecipeUrl` and `recipeInfoUrl` or something similar